### DISCUSSION TITLE
Send current line command

### DISCUSSION THREAD

**Discussion by JaimePalM** [R-nvim/R.nvim#230] [Q&A]
When I type the <LocalLeader>d command to send one line (the current under the cursor), it sends the whole file (through Rnvim.source()). Anyone know what is happening?

**Comment by PMassicotte** [comment]
What is the *whole find*?

**Reply by JaimePalM** [reply]
> What is the _whole find_?

Sorry, the whole file

**Comment by PMassicotte** [comment]
What happens if you try 

```lua
:lua require('r.send').line(false)
```

**Reply by JaimePalM** [reply]
It doesn't change nothing. If it helps, when I start sending commands, it works fine. The problem is when I spent some time working and, at some point, every <LocalLeader>d command makes execute the whole file or more lines than the current at the cursor (in this case, always the same group of lines).

**Comment by jalvesaq** [comment]
Some problems have been solved by deleting the `nvim-treesitter` directory to force updating all parsers. For those using `lazy.nvim`:

```
cd ~/.local/share/nvim/lazy/
rm -rf nvim-treesitter
```
